src/main/resources/jenkins/plugins/ui_samples/Tooltips/index.jelly

```diff
@@ -0,0 +1,36 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:x="jelly:xml">
+  <s:sample>
+    <div class="sample-remote app-component-sample" data-sample="simple-button.html" />
+    <pre>
+      <code class="sample-remote language-html" data-sample="simple-button.html" />
+    </pre>
+
+    <h2>Showing HTML in tooltips</h2>
+    <div class="app-component-sample">
+      <j:set var="tooltip">
+        <img src="${imagesURL}/svgs/logo.svg" />
+      </j:set>
+      <button html-tooltip="${tooltip}" class="jenkins-button jenkins-button--primary">
+        Hover over me
+      </button>
+    </div>
+    <pre>
+      <code class="language-xml">
+        <xmp>
+          &lt;j:set var="tooltip"&gt;
+            &lt;img src="${imagesURL}/svgs/logo.svg" /&gt;
+          &lt;/j:set&gt;
+          <button html-tooltip="$${tooltip}" class="jenkins-button jenkins-button--primary">
+            Hover over me
+          </button>
+        </xmp>
+      </code>
+    </pre>
+    <p class="warning">
+      Make sure that text is <a href="https://en.wikipedia.org/wiki/HTML_sanitization">sanitized</a> before displaying it inside of <code>html-tooltip</code> as this
+      can lead to XSS attacks. Avoid user-controlled data like agent names or descriptions.
+    </p>
```

Comment:
@Wadeck / @daniel-beck anything you would like to add on possible XSS if you aren't careful with embedding HTML? Or do you think this is enough?

Reply:
That's great to have such a warning. Ironically, the agent name is (normally) safe, as we forbid all the dangerous characters by using [checkGoodName](https://github.com/jenkinsci/jenkins/blob/d1f1839375f047f45ad4b545e77eb6f49b0629b6/core/src/main/java/jenkins/model/Jenkins.java#L4192)

The `DisplayName` s in general are not safe (as the descriptions).

We could add a link to https://www.jenkins.io/doc/developer/security/xss-prevention/ which explain multiple points on the XSS prevention.

Reply:
added in https://github.com/jenkinsci/ui-samples-plugin/pull/14/commits/dfc8dbbc64e6c08d759a63889015d2029262b2f6